src/pages/credits/edit/Edit.tsx

```diff
@@ -181,6 +182,8 @@ export default function Edit() {
               endpoint="/api/v1/live_preview?entity=:entity"
             />
           )}
+
+          <RemoveLogoCTA />
```

Comment:
Is there a reason we didn't bake this directly into the InvoicePreview component, or the component one-step lower?